doc/examples/features_detection/plot_blob.py

```diff
@@ -36,13 +36,14 @@
 detected accurately. See :py:meth:`skimage.feature.blob_doh` for usage.
 """
 
+from math import sqrt
 from skimage import data
 from skimage.feature import blob_dog, blob_log, blob_doh
-from math import sqrt
 from skimage.color import rgb2gray
 
 import matplotlib.pyplot as plt
```

Comment:
I've worked out a convention for myself to split the imports into several blocks in the following order:

- standard modules shipped within Python distribution (`math`, `argparse`, `os`, etc);
- PyData stack (in order: `numpy`, `scipy`, `matplotlib`, `skimage`, etc);
- other trusted modules (`pydy`, `nipy`, whatever);
- thrid party modules: `opencv`, `tensorflow`, etc.

IMO, this order makes more sense. Let me know if you insist to change this, I don't mind.